README.md

```diff
@@ -0,0 +1,41 @@
+# AlmaLinux Errata Email Notifications
+This script checks the public errata json files for AlmaLinux distributions and sends notification emails with every new update.
+
+## Requirements
+
+### Dependencies
+The script does not use external libraries, only Python's built-in modules.
+We used Python 3.8 when creating the script but should work with any Python 3.x version.
+
+### Gmail Application Password
+In order for the script to run, you have to provide a [Gmail Application Password](https://support.google.com/accounts/answer/185833).
```

Comment:
Does it work for accounts with 2FA?

Reply:
You can't get an **Application Password** if you don't have 2FA enabled in your Gmail account. You can read more information about it [here](https://support.google.com/accounts/answer/185833?hl=en)